src/System.Memory/src/CompatibilitySuppressions.xml

```diff
@@ -1,8 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!-- https://learn.microsoft.com/en-us/dotnet/fundamentals/package-validation/diagnostic-ids -->
-<Suppressions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
-  <Suppression>
-    <DiagnosticId>PKV006</DiagnosticId>
-    <Target>.NETStandard,Version=v1.1</Target>
```

Comment:
@ViktorHofer @ericstj I don't understand this file removal. The PKV006 error says that this tfm is dropped. We did that on purpose, so we want to suppress the error. Why is the file not necessary anymore?

Reply:
Oh I just saw the other PR description: https://github.com/dotnet/maintenance-packages/pull/68

> Package Validation emitted unnecessary suppression errors because the TFMs are already excluded in packaging.targets.

I assume it's the same case here. Correct?

Reply:
You've excluded these in https://github.com/dotnet/maintenance-packages/blob/4aabc1a42c1decb9fd1b2ce389267728ea8e8587/eng/packaging.targets#L12-L22
which makes API compat not even run on those frameworks.